src/Ookii.Dialogs.WinForms/InputDialogForm.cs

```diff
@@ -67,6 +67,36 @@ public bool UsePasswordMasking
             set { _inputTextBox.UseSystemPasswordChar = value; }
         }
 
+        public bool MultiLine
+        {
+            get { return _inputTextBox.Multiline; }
+            set {
+                if (value != _inputTextBox.Multiline)
```

Comment:
Here we can invert the `if` to have an [early return and reduce nesting](https://softwareengineering.stackexchange.com/questions/18454/should-i-return-from-a-function-early-or-use-an-if-statement).

```csharp
if (value == _inputTextBox.Multiline) return;

// ...
```